### ISSUE TITLE
NVDA does not announce mail search results in search view

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#9840]
**To Reproduce**
1. Enable NVDA
2. Type search query, go into search view to see results
3. See that NVDA just announces "clickable item"

**Notes**
It likely happens because we inhibit the announcement for mail row items by default to override it with our own in mail view but it probably does not work for search row. We need to investigate.
